### PR TITLE
 feat: add pre-commit and mergebot configuration 

### PR DESCRIPTION
**Pull request by makkes**

### CODE REVIEW

**Comment by makkes** [comment]
Please don't merge, yet. I want to create a TC config that runs this, first.

**Review by takirala — APPROVED** [review]

File: mergebot-config.json [hunk 1]
```diff
@@ -5,21 +5,13 @@
   },
   "test": true,
   "backport": true,
-  "update_jira_on_merge": {
```
Comment (gracedo) [inline mergebot-config.json:8]: why remove the jira configs?

**Review by gracedo — APPROVED** [review]

**Comment by gracedo** [comment]
@mesosphere-mergebot backport release-2.1

**Comment by gracedo** [comment]
@mesosphere-mergebot backport release-2.1

**Pull request merged**